py/server/tests/test_vectorization.py

```diff
@@ -251,6 +251,13 @@ def my_sum(*args):
         result = source.update(f"X = my_sum({','.join(cols)})")
         self.assertEqual(len(cols) + 1, len(result.columns))
 
+    def test_enclosed_by_parentheses(self):
+        def sinc(x) -> np.double:
+            return np.sinc(x)
+
+        t = empty_table(100).update(["X = 0.1 * i", "SincXS=((sinc(X)))"])
+        self.assertEqual(t.columns[0].data_type, dtypes.double)
```

Comment:
```suggestion
        def sinc(x) -> np.double:
            return np.sinc(x)

        t = empty_table(100).update(["X = 0.1 * i", "SincXS=((sinc(X)))"])
        self.assertEqual(t.columns[0].data_type, dtypes.double)
        
       def sinc2(x):
            return np.sinc(x)

        t = empty_table(100).update(["X = 0.1 * i", "SincXS=((sinc2(X)))"])
        self.assertEqual(t.columns[0].data_type, ***PyObject)

```

Should have a second test that non-annoted functions return PyObjects and not java Objects.

Reply:
Does there need to be any test to make sure that these two cases still vectorized?  I assume yes.